typescript/web/src/connectors/apollo-client/cache-config.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  InMemoryCacheConfig,
+  FieldFunctionOptions,
+  FieldReadFunction,
+  FieldMergeFunction,
+} from "@apollo/client";
+import type { QueryImagesArgs, Image } from "@labelflow/graphql-types";
+
+export type ImagesQueryCache = {
+  images: Image[];
+  totalCount: number | undefined;
+};
+
+const readQueryImages: FieldReadFunction<ImagesQueryCache, Image[]> = (
+  existing,
+  options: FieldFunctionOptions<QueryImagesArgs, Record<string, any>>
+) => {
+  // A read function should always return undefined if existing is
+  // undefined. Returning undefined signals that the field is
+  // missing from the cache, which instructs Apollo Client to
+  // fetch its value from your GraphQL server.
+  if (existing === undefined) {
+    return undefined;
+  }
```

Comment:
```suggestion
  if (existing === undefined) return undefined;
```

Reply:
I thought we enforced the use of curly brackets ?

Reply:
I do this (**only for simple one-liner early returns**) since my arrival and never heard it was forbidden 😉